project/GenerateOneToManies.scala

```diff
@@ -84,6 +85,9 @@ class OneToManies${n}SQL[A, $bs, E <: WithExtractor, Z](
     extends SQL[Z, E](statement, rawParameters)(SQL.noExtractor[Z]("one-to-many extractor(one(RS => A).toManies(RS => Option[B1]...)) is specified, use #map((A,B) =>Z) instead."))
     with AllOutputDecisionsUnsupported[Z, E] {
 
+  def copy(statement: String = statement, rawParameters: Seq[Any] = rawParameters, one: WrappedResultSet => A = one, $resultSetToOptionsWithDefaultValue, extractor: (A, $seq) => Z = extractor): OneToManies${n}SQL[A, $bs, E, Z] = {
```

Comment:
Ah,  sorry for my misleading wording. The `completely` is not an important part for me. It should have been just different.

I wonder if replacing a statement in such a way is really good for the maintainability of your code. Especially, when you need to replace both of statement and parameters, it will be quite tricky and hard to understand for others. Why don't you reuse SQLSyntax for the purpose?